Improve aggregate stock consumption

lomamech opened the pull request:
- Displaying quantity at the beginning of the month, and the sum of
  stocks exits and stocks entries

closes #5668

lomamech committed:
improve(Aggregate stock Consumption)
- Displaying quantity at the beginning of the month, and the sum of
  stocks exits and stocks entries

closes #5668

lomamech committed:
Test: Commented out of a test due to lack of necessary data

lomamech committed:
Clean and sanitaze code

jniles commented on server/controllers/stock/index.js:
```diff
@@ -909,6 +910,129 @@ async function listLotsDepot(req, res, next) {
+
+  try {
+    const sqlGetMonthlyStockEntry = `
+      SELECT BUID(l.inventory_uuid) AS inventory_uuid, BUID(sm.lot_uuid) AS lot_uuid, 
```
I'm going to try to combine these SQL queries for efficiency.

jniles committed:
chore(stock): improve aggregate consumption

Makes the following improvements to aggregate stock consumption:

1. Uses a single SQL query instead of two to get the entries/exits.
2. Uses `text-right` on the columns with numbers.

jniles commented on client/src/modules/stock/aggregated_consumption/aggregated_consumption.js:
```diff
@@ -145,6 +150,11 @@ function StockAggregatedConsumptionController(
       displayName : 'TABLE.COLUMNS.CONSUMPTION',
       headerCellFilter : 'translate',
       cellTemplate : 'modules/stock/aggregated_consumption/templates/lot_aggregate.tmpl.html',
+    }, {
```
These look good!  I'm going to put `text-right` on them so they can be summed easily in a user's head.

jniles reviewed (commented):
These are good additions.  The UI is a bit cluttered now, but we need these values displayed.  We can make the UX nicer later.
